jar.go

```diff
@@ -396,6 +396,34 @@ func (j *Jar) deleteExpired(now time.Time) {
 	}
 }
 
+// RemoveAll removes any cookies from the jar that were set for the given host.
+func (j *Jar) RemoveAll(host string) {
+	now := time.Now().Add(-1 * time.Second)
```

Comment:
"now" isn't a great name for something that's actually a second in the past.
I'd suggest naming it "expiry" and declaring it closer to where it's used
(just before the for loop)